integration-tests/rest-tests-jaxrs/src/test/groovy/org/hawkular/metrics/rest/ExpressionTagQueryITest.groovy

```diff
@@ -170,6 +170,30 @@ class ExpressionTagQueryITest extends RESTTest {
         headers: [(tenantHeaderName): tenantId])
       assertEquals(200, response.status)
       assertMetricListById(response.data, 'm3', 'm4', 'm5', 'm6')
+
+      response = hawkularMetrics.get(path: "metrics",
+        query: [tags: "a.b"],
+        headers: [(tenantHeaderName): tenantId])
+      assertEquals(200, response.status)
+      assertMetricListById(response.data, 'm6')
+
+      response = hawkularMetrics.get(path: "metrics",
+        query: [tags: "a.b = c.d"],
+        headers: [(tenantHeaderName): tenantId])
+      assertEquals(200, response.status)
+      assertMetricListById(response.data, 'm6')
+
+      response = hawkularMetrics.get(path: "metrics",
+        query: [tags: "a.b = 'c.d'"],
+        headers: [(tenantHeaderName): tenantId])
+      assertEquals(200, response.status)
+      assertMetricListById(response.data, 'm6')
+
+      response = hawkularMetrics.get(path: "metrics",
+        query: [tags: "a.b = 'c.*'"],
```

Comment:
The intention here was to match anything that starts with `c`. 

This is a negative test to make sure that dot is still interpreted correctly for the complex value part of the ANTLR syntax. I added the dot only to the simple value parsing, so wanted to make sure that complex values are still parsed correctly.